packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.routing.js

```diff
@@ -0,0 +1,66 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('octavia-load-balancer.loadbalancers', {
+    url: '/loadbalancers',
+    component: 'octaviaLoadBalancerListing',
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .getAsync('resources')
+        .then((resources) =>
+          resources.length === 0 ? 'octavia-load-balancer.onboarding' : false,
+        ),
+    resolve: {
+      breadcrumb: () => null,
+      goToListingPage: /* @ngInject */ ($state) => () =>
+        $state.go('octavia-load-balancer.loadbalancers', {
+          displayCreationBanner: true,
+        }),
+      goToCreatePage: /* @ngInject */ ($state, atInternet) => () => {
+        atInternet.trackClick({
+          name: `${TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX}::add`,
+          type: 'action',
+        });
+        $state.go('octavia-load-balancer.create');
+      },
+      goToDashboardPage: /* @ngInject */ ($state, atInternet) => (
+        { id, region },
+        isFromEllipsis,
```

Comment:
Use appropriate name. `isFromEllipsis` does not sound good to me.

Reply:
To me it was appropriate because the tracking should be only triggered, when we navigate to dashboard, if it has been triggered from ellipsis link. But ok I changed it to "useTracking". Is it ok?